test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/kubelet-binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: kubelet-cluster-admin
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
@deads2k Never mind, I just figured out the rbac bootstrapping scheme after having a look at the bootstrap policy cluster roles and role-bindings.
I still have one question though, when you want the e2e kubelet-binding to behave just like the binding for system:nodes in the bootstrappolicy, why don't you change the role in [kubelet-binding.yaml](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/e2e-rbac-bindings/kubelet-binding.yaml) from `cluster-admin` to `system:node`?